src/pages/index.js

```diff
@@ -73,25 +115,124 @@ addCardButton.addEventListener("click", () => {
   formValidators["card-add-form"].resetValidation();
 });
 
+//Updating profile information
 function handleProfileFormSubmit(inputValues) {
-  userProfileInformation.setUserInfo(inputValues);
-  profileFormPopup.close();
+  profileFormPopup.setLoading(true);
+  api
+    .updateUserInformation(inputValues)
+    .then((result) => {
+      return result.json();
```

Comment:
`      return result.json();
`

It's not needed in `index.js`


* ` .then(this._checkResponse)` should be in each method to check if the response is `ok`

And it should be in inside `Api`



**Please, correct it everywhere**